packages/story-editor/src/app/layout/useZoomSetting.ts

```diff
@@ -29,7 +29,7 @@ import { themeHelpers } from '@googleforcreators/design-system';
  * Internal dependencies
```

Comment:
The `themeHelpers` from the `design-system` above is used only for `themeHelpers.SCROLLBAR_WIDTH` constant, nothing else.
It's a bit much to type the whole `themeHelpers` within this PR since it would probably require typing `theme`, too. 

We could mark this PR blocked by #12659 or we could temporarily export the `SCROLLBAR_WIDTH` as a separate constant from somewhere and use that directly instead of `themeHelpers`. A constant isn't a helper anyway really, I guess. 

Thoughts? @barklund @swissspidy